apps/console/src/features/applications/components/settings/attribute-management/advance-attribute-settings.tsx

```diff
@@ -169,12 +182,39 @@ export const AdvanceAttributeSettings: FunctionComponent<AdvanceAttributeSetting
         return claimURI;
     });
 
+    interface AdvanceAttributeSettingsErrorValidationInterface {
+
+        sectorIdentifierURI?: string;
+    }
+    function validateForm(values: Record<string, any>): AdvanceAttributeSettingsErrorValidationInterface {
+
+        const errors: AdvanceAttributeSettingsErrorValidationInterface = {
```

Comment:
```suggestion
    function validateForm(values: Record<string, any>): AdvanceAttributeSettingsErrorValidationInterface {
        const errors: AdvanceAttributeSettingsErrorValidationInterface = {
```
For consistency and to align with modern TypeScript and React coding conventions, refactor this to use the arrow function syntax.
Ex:
```
const validateForm = (values: Record<string, any>): AdvanceAttributeSettingsErrorValidationInterface => { ... }
```

Reply:
fixed with [4838970](https://github.com/wso2/identity-apps/pull/4284/commits/4838970b7dcb77e09e551d9946f3a84075499656)